rpcs3/util/media_utils.cpp

```diff
@@ -263,7 +263,7 @@ namespace utils
 			}
 
 			// Find decoder
-			av.codec = avcodec_find_decoder(stream->codecpar->codec_id);
+			av.codec = const_cast<AVCodec*>(avcodec_find_decoder(stream->codecpar->codec_id));
```

Comment:
just make codec const in the header.
I think you can omit the av_free in that case